src/serac/physics/boundary_conditions/boundary_condition_helper.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2019-2021, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include "serac/physics/boundary_conditions/boundary_condition_helper.hpp"
+
+namespace serac::mfem_ext {
+
+void GetEssentialTrueDofsFromElementAttribute(
+    const mfem::ParFiniteElementSpace &fespace,
+    const mfem::Array<int> &elem_attr_is_ess, 
+    mfem::Array<int> &ess_tdof_list, 
+    int component)
+{
+    mfem::Array<int> ess_dofs, true_ess_dofs;
+
+    GetEssentialVDofsFromElementAttribute(fespace, elem_attr_is_ess, ess_dofs, component);
+    fespace.GetRestrictionMatrix()->BooleanMult(ess_dofs, true_ess_dofs);
+    fespace.MarkerToList(true_ess_dofs, ess_tdof_list);
+}
+
+static void mark_dofs(const mfem::Array<int> &dofs, mfem::Array<int> &mark_array)
```

Comment:
This casing is inconsistent.  It should be `MarkDofs`.

Reply:
I copied this function directly from mfem. I left lowercase since `GetEssentialTrueDofsFromElementAttribute` could eventually be a public `mfem::ParFiniteElementSpace` method and would use the `mark_dofs` function that's already in mfem. What do you think? Should I leave it lowercase so that it's compatible with mfem in the future, or change to `MarkDofs`?

Reply:
I guess leave it then.  I thought their functions were always CamelCase...

Reply:
<shrugs> The point of the `mfem_ext` is to match their style. So let's keep it like it is then.